src/test/java/org/fcrepo/camel/integration/FcrepoContainerPreferIT.java

```diff
@@ -184,20 +184,6 @@ public void configure() {
                         "/rdf:RDF/rdf:Description/ldp:contains", ns)
                     .to("mock:filter");
 
-                from("direct:preferHeadersCheckServerManaged")
-                    .to(fcrepo_uri)
-                    .filter().xpath(
-                        "/rdf:RDF/rdf:Description/fedora:uuid", ns)
-                    .to("mock:filter");
-
-                from("direct:includeServerManaged")
-                    .to(fcrepo_uri + "?preferInclude=ServerManaged")
-                    .filter().xpath(
-                        "/rdf:RDF/rdf:Description/fedora:uuid", ns)
-                    .to("mock:filter")
-                    .to(fcrepo_uri)
-                    .to("mock:container");
-
```

Comment:
So the issue with this is that on lines 113 and 133, the camel routes are still calling "direct:preferHeadersCheckServerManaged" and "direct:includeServerManaged". Having deleted those routes, the tests fail. I would recommend restoring these lines, changing `fedora:uuid` to any other server managed property (e.g. fedora:createdBy or fedora:lastModified).